docs-overrides/main.html

```diff
@@ -448,7 +442,7 @@ <h3 class="info-item__title">SedonaSpark</h3>
               <h3 class="info-item__title">SedonaFlink</h3>
               <div class="info-item__description editor">Real-time spatial analytics using Apache Flink.</div>
               <div class="info-item__cta">
-                <a href="#" class="btn-link">
+                <a href="https://sedona.apache.org/latest/setup/flink/install-scala/" class="btn-link">
```

Comment:
Same here. There are a few other places as well.

Please don't use absolute paths, use relative path instead.

I believe it will look like "setup/overview/". Please try locally.